AlphaWallet/AlphaWallet.entitlements

```diff
@@ -2,24 +2,24 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
```

Comment:
Can you see if you can restore this file and not generate a diff?

Reply:
> Can you see if you can restore this file and not generate a diff?

it seems like i fixed it, but i am not sure.